server.js

```diff
@@ -0,0 +1,11 @@
+const http = require('http');
+
+
+const port = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+    res.writeHead(301, {Location: 'https://firefox-test-tube.herokuapp.com/'});
+    res.end();
+});
+
+server.listen(port, () => console.log(`Listening on port ${port}...`));
```

Comment:
I wonder if we can also redirect if the user came in to a specific experiment and send them to the right one?

Reply:
That's a good point. Dammit, Rob. 😆